components/icons/README.md

```diff
@@ -0,0 +1,100 @@
+# Icons
```

Comment:
Most of this is somewhat borrowed from the existing icon documentation. I removed everything about Sass, Polymer iconsets and just referencing SVGs using native `<img>` elements.